### ISSUE TITLE
llvm flag can cause compilation issues

### ISSUE THREAD

**Issue by HuwCampbell** [SeanRBurton/spaceprobe#1]
G'day,

Having the -llvm flag in your cabal file can lead to compilation problems when trying to use multiple versions of GHC.

I don't see anywhere that you're actually using llvm features, so it would be nice if it were removed and up to the user whether they should use llvm.

**Comment by SeanRBurton** [comment]
Yes, thank you. I've been meaning to get around to this for a while.
